README.md

```diff
@@ -6,25 +6,22 @@ Check it out in action at [shuttles.rpi.edu](https://shuttles.rpi.edu).
 
 ## Setting Up
 
-1. Install Go
-2. `go get github.com/wtg/shuttletracker`
-3. `govendor sync`
-4. Make sure you have npm, bower, golang and mongodb installed
-5. Run `bower install` inside shuttle tracking directory to install dependencies listed in bower.json
-6. Rename conf.json.sample to conf.json
+1. Install Golang (https://golang.org/doc/install)
+2. Run `go get github.com/wtg/shuttletracker`
+3. Run `govendor sync`
```

Comment:
Should we include instructions for installing `govendor`?

i.e. `go get -u github.com/kardianos/govendor`

Reply:
Definitely. Totally overlooked that.